lib/krikri/engine.rb

```diff
@@ -75,6 +75,11 @@ def configure_blacklight!
     initializer :rdf_repository do
       Krikri::Repository =
         RDF::Marmotta.new(Krikri::Settings['marmotta']['base'])
```

Comment:
Would this be better as `.fetch` so you have proper key errors if the settings are wonky?